src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/MethodConfigurationComponent.scala

```diff
@@ -140,6 +142,28 @@ trait MethodConfigurationComponent {
       }
     }
 
+    object DeleteMethodConfiguration extends RawSqlQuery {
+      val driver: JdbcDriver = MethodConfigurationComponent.this.driver
+
+      def deleteAction(workspaceId: UUID) = {
+        val tables: Seq[SQLActionBuilder] = Seq(sql"METHOD_CONFIG_INPUT", sql"METHOD_CONFIG_OUTPUT",
+          sql"METHOD_CONFIG_PREREQ")
+
+        def baseQuery(table: SQLActionBuilder) = {
+          concatSqlActions(sql"delete t from ", table,
```

Comment:
Ah, this is so much better, thanks! I was getting stuck trying to do `$table` earlier